### PR TITLE
Task/WA-147: Design Safe Tapis v3 GiD

### PR DESCRIPTION
**Pull request by asimregmi**
[WA-147](https://tacc-main.atlassian.net/browse/WA-147)

Tested and works in dev and prod tenant: https://dev.cep.tacc.utexas.edu/workbench/applications/GiD

~"containerImage": "tapis://cloud.data/corral/tacc/aci/CEP/applications/v3/GiD/GiD_15.0.1.zip"~

"containerImage": tapis://cloud.data/corral/tacc/aci/CEP/applications/v3/interactive-template/interactive.zip

### CODE REVIEW

File: applications/GiD/tapisjob_app.sh [hunk 1]
```diff
@@ -0,0 +1,211 @@
+set -x
```
Comment (rstijerina) [inline applications/GiD/tapisjob_app.sh:1]: Is this file being used?

Comment (asimregmi) [reply applications/GiD/tapisjob_app.sh:1]: No, it's using the one on this location --> corral/tacc/aci/CEP/applications/v3/interactive-template/interactive.zip
I will remove this one.

File: applications/GiD/app.json [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                    "description": "Append host $PATH to Singularity container default $PATH.",
+                    "inputMode": "FIXED",
+                    "notes": {
+                        "isHidden": true
+                    }
+                }
+            ],
+            "archiveFilter": {
+                "includes": [
+                    "inputDirectory"
```
Comment (rstijerina) [inline applications/GiD/app.json:75]: Do we want to archive the inputDirectory?

Comment (asimregmi) [reply applications/GiD/app.json]: Removed inputDirectory from archive

**Review by rstijerina — APPROVED** [review]

**Pull request merged**